src/Chalkboard/Chalkboard.test.js

```diff
@@ -86,47 +73,72 @@ describe('Chalkboard', () => {
     });
 
     it('should increase score when an answer is correct', () => {
-      const { getByText, debug } = render(<Chalkboard />);
+      const { getByText } = render(<MemoryRouter><Provider store={store}><Chalkboard /></Provider></MemoryRouter>);
```

Comment:
Maybe we should try setting our components to a variable in our test suites to save space?